internal/action/generate_test.go

```diff
@@ -178,6 +178,35 @@ func TestGenerate(t *testing.T) { //nolint:paralleltest
 		assert.Contains(t, buf.String(), "Copied to clipboard")
 		buf.Reset()
 	})
+
+	// generate --force foobar w/ pw length set via env variable (42 chars)
+	t.Run("generate --force foobar", func(t *testing.T) { //nolint:paralleltest
+		t.Setenv("GOPASS_PW_DEFAULT_LENGTH", "42")
+
+		if testing.Short() {
```

Comment:
I don't think you need to skip this test in short mode.

Reply:
Thanks for the hint. I've removed the if-condition. 
I'm not exactly sure when to skip tests in short mode and when not. I was just copying the lines from another test. Sorry, should have mentioned that. Is it okay to leave it for the other test in line 193-204, or should I remove it there as well.